packages/patternfly-4/react-core/src/components/Dropdown/DropdownItem.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
```

Comment:
I think we want to decouple the Dropdown component completely from Dropdown here.  So we want to create a new ApplicationLauncher Item component.  If there is any common functionality that can be reused.  Maybe move it into our helper utilities.